static/js/statistics.ts

```diff
@@ -608,31 +85,4 @@ export function toggle_show_students_class_overview(adventure: string) {
   }
 }
 
-export function getRunsOverTime(data: any[], labels: any[]) {
```

Comment:
So is this one

Reply:
In general, the lines that says that they were edited by boryana are safe to delete, and the ones that are more recent are still in use